### PR TITLE
chore: install specific Node version on Mac before building binary

### PR DESCRIPTION
**Pull request by bahmutov**
To match the needed version from `.node-version` file

- [x] probably need to set up the same Node version in multiple jobs for Mac, like build and build-binary
blocked by the code signing error on MAc

```
Frameworks/Electron Framework.framework/Versions/A/Electron Framework: code object is not signed at all
```

Screen Shot 2020-08-11 at 11 31 08 AM

### CODE REVIEW

**Comment by flotwig** [comment]
@bahmutov this runs only on `build` job, right? I think we expect all tests to use the same Node version that was bundled with Electron, so that tests are using the same stdlib as they will be in Electron. So this should run on all test jobs (or at least all of the internal test jobs like server-unit-test, server-e2e-test...)

**Comment by bahmutov** [comment]
yeah, it will require more stuff to happen here, every command needs to do this nonsense

File: circle.yml [hunk 1]
```diff
@@ -925,10 +929,19 @@ jobs:
...
           # won't be set and we will use default version, since we are not going to
           # upload the dev binary build anywhere
-          command: yarn binary-build --platform $PLATFORM --version ${NEXT_DEV_VERSION:-0.0.0-development}
+          # Note 2:
+          # We need to use Node version matching .node-version file
+          command: |
+            export NVM_DIR="/opt/circleci/.nvm"
+            [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+            nvm install v12.14.1
+            nvm alias default v12.14.1
```
Comment (brian-mann) [inline circle.yml:942]: isn't this hard coding the version instead of pulling it from `.node-version`?

**Comment by bahmutov** [comment]
Yup it is hard coding it for now

**Pull request merged**